### PR TITLE
Brian gillet patch 2

### PR DESCRIPTION
**Pull request by brian-gillet**

### CODE REVIEW

File: website/docs/docs/dbt-cloud/deployments/single-tenant.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ id: single-tenant-deployment
...
-A few common reasons for choosing a Single Tenant deployment over the Production SaaS product include:
+A couple common reasons for choosing the Virtual Private dbt plan, which includes a Single Tenant deployment include:
 - A requirement that the dbt Cloud application be hosted in a dedicated VPC that is logically separated from other customer infrastructure
 - A desire for multiple isolated dbt Cloud instances for testing, development, etc
 
-_To learn more about setting up a dbt Cloud Single Tenant deployment, [please contact our sales team](mailto:sales@getdbt.com)._
+_To learn more about the Virtual Private dbt plan, [please contact our sales team](mailto:sales@getdbt.com)._
 
-For more information about the dbt Cloud Single Tenant deployment see the below.
+For more information about the Single Tenant deployment see the below.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud/deployments/single-tenant.md]:
```suggestion
For more information about the Single Tenant deployment:
```

File: website/docs/docs/dbt-cloud/deployments/multi-tenant.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ title: Multi Tenant (SaaS)
 
 The Multi Tenant (SaaS) deployment environment refers to the SaaS dbt Cloud application hosted by dbt Labs. This is the most commonly used deployment and is completely managed and maintained by dbt Labs, the makers of dbt. As a SaaS product, a user can quickly [create an account](https://www.getdbt.com/signup/) and get started using the product. The deployment is hosted in AWS and will always contain the latest software updates and bug fixes.
 
+If your organization requires more advanced security, SSO/RBAC, or additional support, please contact dbt Labs sales (sales@dbtlabs.com) to learn more about our Enterprise plans.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud/deployments/multi-tenant.md]:
```suggestion
If your organization requires advanced security features, SSO/RBAC, or additional support, please contact dbt Labs sales (sales@dbtlabs.com) to learn more about our Enterprise plans. 
```

File: website/docs/docs/dbt-cloud/deployments/deployment-overview.md [hunk 3]
```diff
@@ -5,12 +5,14 @@ title: Overview
...
 :::info Deployments
 
-This section provides an overview of the deployment types available for dbt Cloud.
+This section provides an overview of the deployment types available with specific dbt Cloud plans.
 If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
 
 :::
 
-In general, the dbt Cloud application deployment models fall into two categories: **Multi Tenant** and **Single Tenant**. These deployments are hosted on infrastructure managed by dbt Labs. Both models leverage AWS infrastructure as described in the [Architecture](deployment-architecture) section.
+In general, the dbt Cloud application deployment models fall into two categories: **Multi Tenant** and **Single Tenant**. These deployments are hosted on infrastructure managed by dbt Labs. The Multi Tenant model leverages AWS infrastructure as described in the [Architecture](deployment-architecture) section.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud/deployments/deployment-overview.md]:
```suggestion
The dbt Cloud application deployment models fall into two categories: **Multi-Tenant** and **Single Tenant**. These deployments are hosted on infrastructure managed by dbt Labs. The Multi-Tenant model leverages AWS infrastructure as described in the [Architecture](deployment-architecture) section. 
```

**Comment by matthewshaver** [comment]
Putting this PR aside until we approach Coalesce.  Want to align the language changes here with other product areas as well

**Comment by mirnawong1** [comment]
hey @matthewshaver wanted to flag if this pr was still valid per your comment?

**Comment by mirnawong1** [comment]
hey @matthewshaver is this something that we still need open?

**Comment by matthewshaver** [comment]
Closing due to age - the pages in question have been heavily edited, moved, merged, or removed since this was opened

**Pull request closed without merging**